src/utils/user_utils.test.js

```diff
@@ -131,10 +131,6 @@ describe('user utils', () => {
             assert.deepEqual(filterProfilesMatchingTerm(users, 'some'), [userB]);
         });
 
-        it('should not match by nickname substring', () => {
```

Comment:
Hey! Sorry again for another comment but I feel we shouldn't purge this test altogether. Instead, since now we are doing a wildcard search, I would say rename the test title to `should match by nickname substring` since the functionality has changed.

Reply:
I'm just not a big fan of completely purging a test unless absolutely required. I personally always look to add or modify test to match new behaviour. Thanks!

Reply:
Hi, you're correct. Sorry about that, I thought the test before the one I removed would be sufficient. I just did another commit. Sorry for all the trouble.